api/src/api/users/user_routes.py

```diff
@@ -34,3 +41,30 @@ def user_token(x_oauth_login_gov: dict) -> response.ApiResponse:
     logger.info(message)
 
     raise_flask_error(400, message)
+
+
+@user_blueprint.post("/token/refresh")
+@user_blueprint.output(UserTokenRefreshResponseSchema)
+@user_blueprint.doc(responses=[200, 401])
+@user_blueprint.auth_required(api_jwt_auth)
+@flask_db.with_db_session()
+def user_token_refresh(db_session: db.Session) -> response.ApiResponse:
+    logger.info("POST /v1/users/token/refresh")
+
+    user_token_session: UserTokenSession = api_jwt_auth.current_user  # type: ignore
+    config = get_config()
+    expiration_time = datetime_util.utcnow() + timedelta(minutes=config.token_expiration_minutes)
```

Comment:
Put this in the api jwt auth file, and follow the pattern it has for getting the config (just want to keep that located all in one place as it's a bit weird).